tests/unit/test_errors.py

```diff
@@ -72,6 +72,11 @@ async def test_dynamo_retries_50x(status: int) -> None:
     async def http(request: Request) -> Response:
         return next(responses)
 
-    client = Client(http, creds, "test", throttle_config=StaticDelayRetry(delay=0.01))
+    client = Client(
+        http,
+        creds,
+        "test",
+        throttle_config=StaticDelayRetry(delay=0.01),
+    )
```

Comment:
This format change is odd, because original line was <88 chars.
Do you have `pre-commit` installed in this repo?

Reply:
yes i do have `pre-commit` installed 🤔 will dig in